.npmrc

```diff
@@ -0,0 +1,2 @@
+# Comment out this line to publish to the main registry.
```

Comment:
I'm now thinking this file shouldn't be in git; my mistake I will remove. People who know what they are doing and need to use this can add it.